sdk_container/src/third_party/coreos-overlay/sys-apps/systemd/systemd-256.2.ebuild

```diff
@@ -254,14 +254,11 @@ src_prepare() {
 		"${FILESDIR}/systemd-test-process-util.patch"
 		# Flatcar: Adding our own patches here.
 		"${FILESDIR}/0001-wait-online-set-any-by-default.patch"
-		"${FILESDIR}/0002-networkd-default-to-kernel-IPForwarding-setting.patch"
```

Comment:
To be investigated if the 256 changes are making this patch irelevant or not: https://github.com/systemd/systemd-stable/blob/v256/src/network/networkd-network.c#L470